great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -330,6 +331,11 @@ def _check_input_validation(
                     "passed": False,
                 }
             )
+        if "validate_configuration" not in expectation_instance.__class__.__dict__:
+            passed = False
```

Comment:
And get rid of `passed = False` (since it's already False) and use `"passed": passed` in that sub_message too